vendor.conf

```diff
@@ -124,7 +124,7 @@ github.com/pkg/errors 839d9e913e063e28dfd0e6c7b7512793e0a48be9
 
 # cli
 github.com/spf13/cobra v1.5 https://github.com/dnephin/cobra.git
-github.com/spf13/pflag dabebe21bf790f782ea4c7bbd2efc430de182afd
+github.com/spf13/pflag e916533c8c5c345b1ed8e9db5dcfbf99525828b7 https://github.com/ijc25/pflag.git
```

Comment:
Please open a PR against https://github.com/spf13/pflag to try and get these changes applied upstream